src/ts/components/elements/Application/Application.tsx

```diff
@@ -1506,7 +1506,10 @@ class UnwrappedApplication extends React.PureComponent<ApplicationProps, Applica
     }
 
     private pollopolloDonation = async () => {
-        this.setState({ showPollopolloDonation: true });
+        if(this.props.userType === UserTypes.DONOR) this.setState({ showPollopolloDonation: true });
+        else {
+            routes.registerProducer.path;
```

Comment:
What should this line do? It currently does nothing

Reply:
woops, fixed that on the develop branch 😅